### PR TITLE
Sdk tracking

### PR DESCRIPTION
**Pull request by josephwegner**
Adds a new header for tracking the SDK version.

Oddly, there is no where that programmatically indicates the current API version, so had to define a constant for that :(

### CODE REVIEW

File: src/Client/KeenIOClient.php [hunk 1]
```diff
@@ -7,6 +7,8 @@
 use GuzzleHttp\Client;
 use KeenIO\Exception\RuntimeException;
 
+define('VERSION', '2.5.7');
```
Comment (stof) [inline src/Client/KeenIOClient.php]: please don't define a global constant. This is likely to conflict with any library out there, as the name is ultra generic.

You should either use a class constant or a namespaced constant

Comment (josephwegner) [inline src/Client/KeenIOClient.php]: Thanks @stof - good catch. Been awhile since I've done PHP. I will update.

**Comment by josephwegner** [comment]
@stof look better?

**Comment by stof** [comment]
wouldn't it make sense to use the user-agent to pass this info instead ?

**Comment by EmilyShepherd** [comment]
[Purist patch neatness comment] Using 5 commits to do what could be done in 1 is a tad ugly, might be worth considering squashing these together?

**Comment by josephwegner** [comment]
@stof Can't use User-Agent, unfortunately. We are using a standard header across all of our SDKs. Some of them, javascript in particular, rely heavily on extracting useful data from the default client's User-Agent. If we override that with Keen SDK information, it would result in our customer's having less rich data.

**Comment by josephwegner** [comment]
@EmilyShepherd Cleaned up - good feedback.

**Comment by stof** [comment]
@josephwegner yeah, I thought it might have been the case. So fine.

File: src/Client/KeenIOClient.php [hunk 2]
```diff
@@ -39,6 +39,9 @@
  */
 class KeenIOClient extends GuzzleClient
 {
+
+    const VERSION = '2.5.7'
```
Comment (stof) [inline src/Client/KeenIOClient.php]: missing ``;`` which is why the tests are broken

**Pull request merged**